src/test/java/entityAPI/AddressAPI.java

```diff
@@ -0,0 +1,23 @@
+package entityAPI;
+
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import test.ConfigLoader;
+
+import static io.restassured.RestAssured.given;
+
+public class AddressAPI {
+    private static final ConfigLoader properties = ConfigLoader.getInstance();
+    private static final String baseURI = properties.getPropertyValue("baseURI");
+    private static final String ADDRESS_ENDPOINT = properties.getPropertyValue("ADDRESS_ENDPOINT");
+
+    public static Response getAddress(RequestSpecification request) {
```

Comment:
extract a common method for all APIs that will call "spec" on a request object and set baseUri value.
Usually request is more complicated than this, you'll have some authentication stuff too, it makes sense to have it separate